semana-18/FutureBook/src/business/gateways/Post/PostGateway.ts

```diff
@@ -0,0 +1,9 @@
+import { Comment } from '../../entities/Comment';
+import { Post } from './../../entities/Post';
+
+export interface PostGateway {
+    createPost(post: Post): Promise<void>;
+    likePost(userId: string, postId: string): Promise<object>;
+    toComment(comment: Comment): Promise<void>;
```

Comment:
Achei o nome `toComment` meio ruim. Normalmente, usamos o `to` quando estamos falando de passar uma informação **para** alguém/alguma classe/alguma função!
Renomearia para `comment`, `addComment`